src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -973,6 +974,12 @@ private function addAssetMapperSection(ArrayNodeDefinition $rootNode, callable $
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('importmap_integrity_algorithms')
+                            ->info('Array of algorithms used to compute importmap resources integrity.')
+                            ->beforeNormalization()->castToArray()->end()
+                            ->prototype('scalar')->end()
+                            ->defaultValue([])
+                        ->end()
```

Comment:
I would clearly state _**all**_ of them will be executed for every asset, not just the first one or anything. 

This may have a sensible impact on time and energy used on CI or deployment.

Reply:
Would a second `They will all run for every asset.` sentence be clear enough? Or how would you document it?